packages/ui/src/preloader/Preloader.tsx

```diff
@@ -0,0 +1,13 @@
+import { Skeleton as MuiSkeleton, SkeletonProps } from '@material-ui/lab';
+import styled from 'styled-components';
+
+const StyledSkeleton = styled(MuiSkeleton)`
+  &.MuiSkeleton-text {
+    transform: scale(1, 1);
+    border-radius: 3px;
+  }
+`;
```

Comment:
@umrzoq-toshkentov Can we modify it via theme?

Reply:
@doniyor2109 If I modify via theme, it could affect existing Skeleton component and then can break design. That's why I created separated component itself.

Reply:
@umrzoq-toshkentov I think it is ok if it changes existing components. We should update them in application codebase

Reply:
okay I will update then

Reply:
the main issue will be about skeleton height. I have researched existing component and seen skeleton wrapped with `div` component, and existing skeleton `span` scaled up `60%` of height.
For example if you give <Skeleton height={30} />, Skeleton height will be 18px.

Reply:
fixed overrided with theme